src/neuroglancer/image_user_layer.ts

```diff
@@ -291,37 +291,37 @@ function makeShaderCodeWidget(layer: ImageUserLayer) {
 
 const LAYER_CONTROLS: LayerControlDefinition<ImageUserLayer>[] = [
   {
-    label: 'Resolution (slice)',
+    label: 'Resolution selector (2D)',
     toolJson: CROSS_SECTION_RENDER_SCALE_JSON_KEY,
     ...renderScaleLayerControl(layer => ({
                                  histogram: layer.sliceViewRenderScaleHistogram,
                                  target: layer.sliceViewRenderScaleTarget
                                })),
   },
   {
-    label: 'Blending',
+    label: 'Cross-section blending',
     toolJson: BLEND_JSON_KEY,
     ...enumLayerControl(layer => layer.blendMode),
   },
+  {
+    label: 'Cross-section opacity',
+    toolJson: OPACITY_JSON_KEY,
+    ...rangeLayerControl(layer => ({value: layer.opacity})),
+  },
   {
     label: 'Volume rendering (experimental)',
     toolJson: VOLUME_RENDERING_JSON_KEY,
     ...checkboxLayerControl(layer => layer.volumeRendering),
   },
   {
-    label: 'Resolution (3d)',
+    label: 'Resolution indicator (3D)',
```

Comment:
Per the comment in the other PR, the change to this label may be unnecessary if it is changed to control the number of samples.